pkg/pronom/identifier.go

```diff
@@ -119,7 +119,7 @@ func New(opts ...config.Option) (core.Identifier, error) {
 	pronom = identifier.ApplyConfig(pronom)
 	id := &Identifier{
 		Base:     identifier.New(pronom, config.ZipPuid()),
-		hasClass: config.Reports() != "" && !config.NoClass(),
+		hasClass: !config.NoClass(),
```

Comment:
I believe this is all we need to do here when this is possible with the signature file too?